ci/test/01_base_install.sh

```diff
@@ -6,35 +6,27 @@
 
 export LC_ALL=C.UTF-8
 
-# This script is always run as root, and the functions can be removed and
-# replaced by a bash -c "command" directly, where needed.
-# For now, they are named aliases to aid review with --color-moved=dimmed-zebra.
-CI_EXEC_ROOT () { bash -c "$*"; }
-CI_EXEC()       { bash -c "$*"; }
-export -f CI_EXEC_ROOT
-export -f CI_EXEC
-
 if [ -n "$DPKG_ADD_ARCH" ]; then
-  CI_EXEC_ROOT dpkg --add-architecture "$DPKG_ADD_ARCH"
+  dpkg --add-architecture "$DPKG_ADD_ARCH"
 fi
 
 if [[ $CI_IMAGE_NAME_TAG == *centos* ]]; then
-  ${CI_RETRY_EXE} CI_EXEC_ROOT dnf -y install epel-release
-  ${CI_RETRY_EXE} CI_EXEC_ROOT dnf -y --allowerasing install "$CI_BASE_PACKAGES" "$PACKAGES"
+  ${CI_RETRY_EXE} bash -c "dnf -y install epel-release"
+  ${CI_RETRY_EXE} bash -c "dnf -y --allowerasing install $CI_BASE_PACKAGES $PACKAGES"
```

Comment:
It is basically the same trick described in #27321 to be used for `apt`, `dnf` is basically the Redhat package manager
The other option would be to arrayify PACKAGES and CI_BASE_PACKAGES. I have that in my git stash too, but even though it is a cleaner, it actually adds a couple extra lines when initialising the defaults. I value both cleanness and simplicity, so I cannot choose myself which way is better.

The interesting question is whether I should do the same for pip3 - otherwise the guy who adds a second python package would have to do the same research as me.